adapters/kosovo.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * This code is responsible for implementing all methods related to fetching
+ * and returning data for a Kosovo sources (HTML page)
+ */
+'use strict';
+
+const moment = require('moment-timezone');
+const request = require('request-promise-native');
```

Comment:
Please follow a similar pattern to https://github.com/openaq/openaq-fetch/blob/develop/adapters/airnow-http.js#L11 and include the `REQUEST_TIMEOUT` so that the adapter doesn't hang the entire process. Looks like we lost this for the adapters using `request-promise-native`, I'll fix these separately.